src/pages/RoutesPage.tsx

```diff
@@ -0,0 +1,45 @@
+import { AppShell, Group, Image, NavLink, Title } from '@mantine/core';
+
+import { DashboardIcon, HomeIcon } from '@/icons/icons';
+
+export function RoutesPage() {
+  return (
+    <AppShell
+      header={{ height: 60 }}
+      navbar={{
+        width: 300,
+        breakpoint: 'sm'
+      }}
+      padding={'md'}
+    >
+      <AppShell.Header>
+        <Group h={'100%'} px={'md'}>
+          <Image
```

Comment:
The logo doesn't show for me. Might need to set a `width`

Reply:
Fricking firefox 😆